core/auth/src/main/java/software/amazon/awssdk/auth/credentials/internal/ProfileCredentialsUtils.java

```diff
@@ -46,10 +47,12 @@
 /**
  * Utility class to load {@link #credentialsProvider()} configured in a profile.
  */
-@SdkInternalApi
+@SdkProtectedApi
```

Comment:
Why is this protected?

Reply:
That was a mistake, I fixed it once... but I guess the fix wasn't included... will update it again.